.packit.yaml

```
@@ -0,0 +1,29 @@
+specfile_path: ansible-lint.spec
+actions:
+  create-archive:
+    - "python3 setup.py sdist --dist-dir ."
+    - "sh -c 'echo ansible-lint-$(python3 setup.py --version).tar.gz'"
+  get-current-version:
+    - "python3 setup.py --version"
+  post-upstream-clone:
```

@ssbarnea _I think_ this is the wrong hook to be doing this. There's `fix-spec-file` that seems to be a better fit.
```suggestion
  fix-spec-file:
```

---

`fix-spec-file` action also changes the %autosetup line to make sure the tarball is unpacked and accessed properly which you may not need: it should be fine to use `post-upstream-clone` action for that, depends what works best for you